src/surge-xt/osc/OpenSoundControl.cpp

```diff
@@ -134,13 +134,14 @@ void OpenSoundControl::oscMessageReceived(const juce::OSCMessage &message)
 
         float frequency = message[0].getFloat32();
         int velocity = static_cast<int>(message[1].getFloat32());
+        constexpr float MAX_MIDI_FREQ = 1625.49867722; // 2^(128/12)
```

Comment:
This is the pitch. Need to mult by midi note 0 to get the hz

Reply:
(Copied/pasted from Discord): I see 12543.854* referenced as the max MIDI frequency, but 8.175798915643710529 (MIDI_FREQ_0) * 1625.49867722 (your value) = 13289.75. 

*I can get this same value by passing 127 to Pd's 'mtof', which converts MIDI notes to frequencies.

So, I've set MAX_MIDI_FREQ to 12543.854. Let me know if that's ok.